rust/sample-plugin/src/main.rs

```diff
@@ -70,6 +71,16 @@ impl Plugin for SamplePlugin {
                 let _ = self.capitalize_word(view, iv.end());
             }
         }
+        self.0 += 1;
+        view.update_status_item("my_key", &format!("hello {}", self.0));
```

Comment:
after you get to 100 and remove "my_key", this stops doing anything. Overall I like the example, but maybe it could be tweaked a bit?